vnext/ReactUWP/Views/TextViewManager.cpp

```diff
@@ -22,11 +22,41 @@ namespace uwp {
 class TextShadowNode : public ShadowNodeBase {
   using Super = ShadowNodeBase;
 
- public:
-  TextShadowNode() = default;
+ private:
+  ShadowNode *m_firstChildNode;
+
+ public:  
+  TextShadowNode() {
+    m_firstChildNode = nullptr;
+  };
   bool ImplementsPadding() override {
     return true;
   }
+
+  void AddView(ShadowNode &child, int64_t index) override {
+    if (index == 0) {
+      auto run =
+          static_cast<ShadowNodeBase &>(child).GetView().try_as<winrt::Run>();
```

Comment:
child->GetView()
?

Reply:
has to be ShadowNodeBase to call getView